main.go

```diff
@@ -93,6 +93,11 @@ func main() {
 	}
 
 	// Migrate the schema
+	err = migration.Migrate(db)
+	if err != nil {
+		panic(err.Error())
```

Comment:
@kwk I have written a test case to demonstrate it: https://github.com/kwk/almighty-core/pull/1